templates/components/products/list.html

```diff
@@ -2,9 +2,9 @@
     {{#each products}}
     <li class="product">
         {{#if settings.data_tag_enabled}}
-            {{>components/products/list-item   show_compare=../show_compare show_rating=../settings.show_product_rating theme_settings=../theme_settings customer=../customer event=../event position=(add @index 1)}}
+            {{>components/products/list-item   show_compare=../../show_compare show_rating=../settings.show_product_rating theme_settings=../theme_settings customer=../customer event=../event position=(add @index 1)}}
```

Comment:
Can you explain this change? Why is it required for `show_compare` but not for others like `show_rating`?

Reply:
@mattolson Entering an if block adds another level to the context, so the parent becomes ../../foo 
Looking on the grid display, I do see that the other values also will be available in the same context as show_compare so they will also should to be modified. I also tested @index and this variable is available and no change needed for it.